src/app/profile/@createNftModal/loading.tsx

```diff
@@ -0,0 +1,11 @@
+import Modal from "@/components/modal";
+
+const Loading = () => {
+  return (
+    <Modal>
+      <p className="animate-bounce text-xl text-white">Loading ...</p>
```

Comment:
Move this line to a separate component because you copy this row of code many times.